src/aunit/TestRunner.h

```diff
@@ -157,6 +158,18 @@ class TestRunner {
     /** Constructor. */
     TestRunner() {}
 
+    int ellapsedSeconds() const
+    {
+      auto now = std::chrono::high_resolution_clock::now();
+      return std::chrono::duration_cast<std::chrono::seconds>(now - mStartTime).count();
```

Comment:
This commit shouldn't be mixed into this PR. I'll comment in #96 instead.

Reply:
Ho yes, my mistake. Sorry for this.